### PR TITLE
Test

### PR DESCRIPTION
**Pull request by Aparnakrishnan-Vonnue**

### CODE REVIEW

**Comment by Aparnakrishnan-Vonnue** [comment]
Mobile view of Home page completed. Responsive Desktop view in progress.

File: Test/blog.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                    <span>11 Oct 2020</span>
+                    <h2>Home Made Standing Desk Raspberry Pi Controller</h2>
+                    <p>One of my lockdown projects was to make my sit?stand controller smart.
+                       I used a Raspberry Pi to create a custom controller that could control
+                       the motor on my desk and alow me to rise and lower it over the network
+                       as well as via touch-interface.
+                    </p>
+                    <button>#raspberry-pi</button>
+                </div>
+                <div class="four">
```
Comment (vonnue-sijin) [inline Test/blog.html:42]: Provide meaningful class names.

File: Test/index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        <header>
+            <div class="menu">
+                    <a href="#blog">Blog</a>
+                    <a href="#contact">Contact</a>
+                    <span><a href="#email">@aparnakrishnan</a></span>
+            </div>
+        </header>
+            <div class="banner">
+                <h1>Aparna <br> Krishnan </h1>
+                <img src="/images/download.jpeg">
```
Comment (vonnue-sijin) [inline Test/index.html:19]: Provide `alt` tag always.

File: Test/index.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                <p>—— Substack's UI and 1Password just cost me $2,023</p>
+                <p>—— What Saturday on Hacker News Looks Like</p>
+                <p>—— Home Mad Standing desk Raspberry Pi Controller</p>
+                <p>—— Saving Your Initial Reaction</p>
+                <p>—— Django Pagedown Image uploads</p>
+            </div>
+            </div>
+            <footer>
+                <div class="flex">
+                <div class="shortcuts">
```
Comment (vonnue-sijin) [inline Test/index.html:35]: Indent code.

File: Test/index.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+            <div class="post2">
+                <span>25.10.2020</span>
+                <p>What Saturday on Hacker News Looks Like</p>
+            </div>
+            <div class="post3">
+                <span>11.10.2020</span>
+                <p>Home Mad Standing desk Raspberry Pi Controller</p>
+            </div>
+            <div class="post4">
+                <span>06.08.2020</span>
```
Comment (vonnue-sijin) [inline Test/index.html:85]: Is this how we create responsive UIs? Don't duplicate elements.

File: Test/style.css [hunk 5]
```diff
@@ -0,0 +1,237 @@
+/* Variables */
+:root{
+    --primary: #07691c;
+    --secondary:#2b2b2b; 
+    --teritiary: #b5b5b5;
+    --quarternary: #ffe8dc;
```
Comment (vonnue-sijin) [inline Test/style.css:6]: :+1: 
Good you used CSS variables, but try to come up with better naming.

File: Test/style.css [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+}
+.posts .post4 span:hover{
+    color: var(--primary);
+}
+.posts .post5 span{
+    margin-top: 13px;
+}
+.posts .post5 span:hover{
+    color: var(--primary);
+}
```
Comment (vonnue-sijin) [inline Test/style.css:227]: Can't we style all this in a single go?

File: Test/style1.css [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+    border-bottom:1px solid #b5b5b5;
+}
+.content .four button{
+display: flex;
+flex-direction: row;
+justify-content: space-evenly;
+display: flex;
+flex-direction: row;
+justify-content: space-between;
+background-color: var(--quarternary);
```
Comment (vonnue-sijin) [inline Test/style1.css:90]: Indent code.

**Review by vonnue-sijin — COMMENTED** [review]
Give better file name

**Comment by Aparnakrishnan-Vonnue** [comment]
Thank you sir for reviewing. After started web cloning task, got an idea of
naming of files. Reviews are noted. Trying to do better in coming tasks.
Will be overcoming all the flaws in coming tasks.

Thanks & Regards
Aparna

**Pull request closed without merging**